lib/src/usage_impl_io.dart

```diff
@@ -29,7 +32,12 @@ class AnalyticsIO extends AnalyticsImpl {
             new IOPostHandler(),
             applicationName: applicationName,
             applicationVersion: applicationVersion,
-            analyticsUrl: analyticsUrl);
+            analyticsUrl: analyticsUrl) {
+    final String locale = getPlatformLocale() ?? '';
+    if (locale != null) {
```

Comment:
yup; remove the `??` operator - we only send the locale now if we can determine one